recipes/icon/build.sh

```diff
@@ -0,0 +1,132 @@
+#!/bin/env bash
+
+# Abort script upon untested failure
+set -e
+
+make Configure name=linux
+
+make CC=${CC}
+
+# Run a small sample of the test suite
+make Samples
+
+# Deploy to icon and bin directories at PREFIX
+mkdir -p ${PREFIX}/bin
+DEPLOY=${PREFIX}/icon
+
+# Install the basics
+make Install dest=${DEPLOY}
+mv ${DEPLOY}/man ${PREFIX}/man
+
+# Put executables onto the path
+(pushd ${PREFIX}/bin && ln -s ../icon/bin/* .)
+
+# Leave the breadcrumbs required to build loadable C functions
+cp ${SRC_DIR}/Makedefs ${DEPLOY}/Makedefs
+mkdir -p ${DEPLOY}/ipl/cfuncs; cp    ${SRC_DIR}/ipl/cfuncs/* ${DEPLOY}/ipl/cfuncs/
+mkdir -p ${DEPLOY}/ipl/data  ; cp    ${SRC_DIR}/ipl/data/*   ${DEPLOY}/ipl/data/
+mkdir -p ${DEPLOY}/ipl/docs  ; cp    ${SRC_DIR}/ipl/docs/*   ${DEPLOY}/ipl/docs/
+mkdir -p ${DEPLOY}/ipl/incl  ; cp    ${SRC_DIR}/ipl/incl/*   ${DEPLOY}/ipl/incl/
+mkdir -p ${DEPLOY}/ipl/packs ; cp -R ${SRC_DIR}/ipl/packs/*  ${DEPLOY}/ipl/packs
+mkdir -p ${DEPLOY}/ipl/procs ; cp    ${SRC_DIR}/ipl/procs/*  ${DEPLOY}/ipl/procs/
+mkdir -p ${DEPLOY}/ipl/progs ; cp    ${SRC_DIR}/ipl/progs/*  ${DEPLOY}/ipl/progs/
+
+# Save cfuncs.u? because we have to omit from the build the build tools
+#   needed to reconstruct it via ipl/cfuncs/Makefile
+cp ${DEPLOY}/lib/cfunc.u? ${DEPLOY}/
+# Removing the ucode from lib would reduce the size of the conda package by
+#   almost half; the cost of this would be a (slightly) prolonged first activation.
+du -sh --apparent-size ${DEPLOY}/lib > ${DEPLOY}/lib/initial_size
+rm ${DEPLOY}/lib/*.u?
+# However, it apparently is better to increase the size of the package rather
+#   than defering this task to the activation; the reason for removing lib/*.u?
+#   above is that the default *.u? includes some files (2 Mb out of 4.6 Mb)
+#   that are not relevant to this conda pkg.
+pushd ${DEPLOY}/ipl/procs
+unset LPATH
+../../bin/icont -usc *.icn; mv *.u? ../../lib
+popd
+du -sh --apparent-size ${DEPLOY}/lib > ${DEPLOY}/lib/final_size
+
+# restore cfuncs.u?
+mv ${DEPLOY}/cfunc.u? ${DEPLOY}/lib/
+
+# Create instructions to make extension libraries in C
+mkdir -p ${DEPLOY}/doc ; cp ${SRC_DIR}/doc/* ${DEPLOY}/doc/
+echo '
+#!/bin/env bash
+
+# To build loadable C functions as described at
+#   https://www.cs.arizona.edu/icon/uguide/cfuncs.htm
+# you will need to add the C compiler and some tools to your environment, e.g.:
+
+    # get the build tools
+    conda install -c conda-forge gcc_linux-64 make sed binutils
+
+    # set the envars that would be set by `conda activate ...`
+    . $CONDA_PREFIX/etc/conda/activate.d/activate-gcc_linux-64.sh
+    . $CONDA_PREFIX/etc/conda/activate.d/activate-binutils_linux-64.sh
```

Comment:
These are unnecessary

Reply:
Indeed, they are ... now.  At one point, I couldn't get it to work without them.  What a puzzle...